TelCo.ColorCoder/Utils/Colors.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace TelCo.ColorCoder.Utils
+{
+   class Colors
+    {
+        public static ColorPair GetColorFromPairNumber(int pairNumber)
```

Comment:
follow the pattern in the code you are changing. 
Add function, class summary to each file each function.

Reply:
Added!